cloudserver/app/Http/Controllers/HomeController.php

```diff
@@ -43,9 +90,65 @@ public function index()
 						->where('id', $user_access->device_id)
 						->first();
 		}
+		
+
 		return 	view('/home')
 				->with('user_devices', $user_devices)
-				->with('ip', $ip);
-        //return view('home');
+				->with('ip', $ip)
+				->with('raspberrys', $raspberrys);*/
```

Comment:
Should it be an end comment tag on row 98?

Reply:
Removed